metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/HiveConnectorPartitionService.java

```diff
@@ -202,6 +202,13 @@ public int getPartitionCount(
                     filteredPartitionList.add(partition);
                 }
             });
+            if (sort != null) {
+                if (sort.getOrder() == SortOrder.DESC) {
+                    Collections.sort(filteredPartitionList, Collections.reverseOrder());
```

Comment:
I still think you should use the comparator here cause it'll bring more flexibility if we ever want to implement something else than just relying on whatever the compareTo has. I guess for now though it's a change either way so up to you